lib/mongo.ex

```diff
@@ -741,17 +747,42 @@ defmodule Mongo do
           false -> {:error, error}
         end
     end
-
   end
 
   @doc false
   @spec exec_command(GenServer.server, BSON.document, Keyword.t) :: {:ok, BSON.document | nil} | {:error, Mongo.Error.t}
   def exec_command(conn, cmd, opts) do
+    cmd
+    |> Enum.into(%{})
+    |> trace_start()
+    
     with {:ok, _cmd, {doc, event}} <- DBConnection.execute(conn, %Query{action: :command}, [cmd], defaults(opts)),
          {:ok, doc} <- check_for_error(doc, event) do
+      trace_end(event)
+
       {:ok, doc}
     end
+  end
+
+  defp trace_start(cmd) do
+    unless Mix.env() == :test do
+      measurements = %{
+        start: DateTime.utc_now |> DateTime.to_string
```

Comment:
any need to make it a string? Actually, why do we need a `start` at all if you get duration from mongo event?